proto/iotics/api/twin.proto

```diff
@@ -70,7 +87,7 @@ message ListAllTwinsResponse {
 
   // Payload of listed twins.
   message Payload {
-    repeated Twin twins = 1;
```

Comment:
Is `message Twin {` still used anywhere?

Reply:
Yes, it's used in the DescribeTwinResponse

Reply:
👍